manifests/init.pp

```diff
@@ -64,6 +67,7 @@
 
   $package_names = $katello::params::package_names,
   $enable_ostree = $katello::params::enable_ostree,
+  $max_keep_alive = $katello::params::max_keep_alive,
```

Comment:
Btw, not sure what kafo does or if you need type:integer as well in the documentation.